src/dc_chat.c

```diff
@@ -259,6 +259,46 @@ char* dc_chat_get_profile_image(const dc_chat_t* chat)
 }
 
 
+/**
+ * Get a color for the chat.
+ * For 1:1 chats, the color is calculated from the contact's email address.
+ * Otherwise, the chat name is used.
+ * The color can be used for an fallback avatar with white initials
+ * as well as for headlines in bubbles of group chats.
+ *
+ * @memberof dc_chat_t
+ * @param chat The chat object.
+ * @return Color as 0x00rrggbb with rr=red, gg=green, bb=blue
+ *     each in the range 0-255.
+ */
```

Comment:
You mean from `0` to `ff`?

Reply:
jupp :)
do you think we should better write it in hex?

Reply:
Nah, it's probably fine :)